commands.go

```diff
@@ -3888,3 +3890,32 @@ func (c cmdable) ACLDryRun(ctx context.Context, username string, command ...inte
 	_ = c(ctx, cmd)
 	return cmd
 }
+
+// ModuleLoadexConfig struct is used to specify the arguments for the MODULE LOADEX command of redis.
+// `MODULE LOADEX path [CONFIG name value [CONFIG name value ...]] [ARGS args [args ...]]`
+type ModuleLoadexConfig struct {
+	Path string
+	Conf map[string]interface{}
+	Args []interface{}
+}
+
+func (c *ModuleLoadexConfig) ToArgs() []interface{} {
```

Comment:
Are we talking about the ToArgs func, or the whole config struct?
If for ToArgs, it was to isolate and be able to unit test that the command signature is generated correctly, since fully testing the command is not possible at the moment.
It requires a module which we need to compile and load.

Reply:
Since it has no meaning to the user, it should be added to the export_test.go file....

Reply:
Changed it, should I resolve?